src/assets/styles/projects copy.scss

```
@@ -1,216 +1,216 @@
-    }
-  }
+	position: relative;
+	display: inline-block;
+	.textcircle {
+		display: block;
+		width: 15.625rem;
+		animation: rotate 7s linear infinite;
+		text {
+			font-size: 2rem;
+			text-transform: uppercase;
+			font: #000;
```

Stylelint (17.4.0)

[error] Unexpected shorthand "font" after "font-size" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)